server/models/Card.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+
+const cardSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  description: {
+    type: String,
+    required: true,
+    unique: true
+  },
+
+  belongsTo : {
+    id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Column'
+    }
+  }
```

Comment:
While I understand the idea here, it's unnecessary we can just do a property named `column` and attach the id directly instead of defining it's relationship model which is already clear. 

Thanks!

Reply:
referencing column directly now.